scripts/cli.py

```diff
@@ -525,7 +526,7 @@ def setup(controller, services, test_mode):
         fail_msg='Waited too long for addons to come up!',
     )
 
-    args = []
+    args = [i for c in [('--config', c) for c in config] for i in c]
```

Comment:
To make it a little easier to read, can we switch this to `args = [f'--config={c}' for c in config]`? (I believe Juju supports the `--config=foo=bar` syntax)

Reply:
sure, this has much better readability.